vscode-extension/package.json

```diff
@@ -49,15 +49,15 @@
     "configuration": {
       "title": "Metals",
       "properties": {
-        "metals.sbt.enabled": {
+        "metals.sbt.diagnostics.enabled": {
           "type": "boolean",
-          "default": false,
-          "description": "EXPERIMENTAL. Use sbt language server to compile on file save and report diagnostics. Requires sbt 1.1.0"
+          "default": true,
+          "description": "EXPERIMENTAL. Enable diagnostics from the sbt server. Requires a running sbt 1.1+ instance."
```

Comment:
No longer experimental.

Reply:
I thought it's experimental because it's "sbt-1.x-only", but I agree that it's stable enough.